pyvat/__init__.py

```diff
@@ -58,12 +56,11 @@
 <http://ec.europa.eu/taxation_customs/vies/faqvies.do>`_.
 """
 
-
 VIES_REGISTRY = ViesRegistry()
+HMRC_REGISTRY = HMRCRegistry()
```

Comment:
The comment "VIES registry instance." should be below line 59 and a new one should be added for HMRC_REGISTRY.